shared_model/interfaces/commands/create_asset.hpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_SHARED_MODEL_CREATE_ASSET_HPP
+#define IROHA_SHARED_MODEL_CREATE_ASSET_HPP
+
+#include "interfaces/common_objects/types.hpp"
+#include "interfaces/primitive.hpp"
+#include "model/commands/create_asset.hpp"
+
+namespace shared_model {
+  namespace interface {
+    /**
+     * Create asset in Iroha domain
+     */
+    class CreateAsset
+        : public Primitive<CreateAsset, iroha::model::CreateAsset> {
+     public:
+      /// Type returned by assetName function
+      using AssetNameType = std::string;
```

Comment:
Why not move it into types?

Reply:
It is asset name, not asset id and it is used only in this function